docs/components/content/ComponentExample.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-  <div class="[&>div>pre]:!rounded-t-none [&>div>pre]:!mt-0">
+  <div class="[&>div>pre]:!rounded-t-none [&>div:last-child]:rounded-t-none [&>div>pre]:!mt-0">
```

Comment:
Does the `[&>div:last-child]:rounded-t-none` still necessary here?

Reply:
Can't remember what it was for, removed in 2023f664